update/Pos/Update/Download.hs

```diff
@@ -171,10 +174,11 @@ downloadHash updateServers h = do
     manager <- liftIO $ newManager tlsManagerSettings
 
     let -- try all servers in turn until there's a Right
+        maxLatency = 2 * 10 ^ (9 :: Int) -- 30 min is enough?
```

Comment:
We tend to avoid using types like `Int` for time units as much as possible. We are currently using [time-units](https://hackage.haskell.org/package/time-units-1.0.0/docs/Data-Time-Units.html) library instead (and later we'll switch to `o-clock`, but that's another story).
It would be better to write `maxLatency = 30 :: Minute` and then convert it to `Int` right before passing to `timeout`.
[o-clock](https://github.com/serokell/o-clock) has `timeout` function with better type, so when we switch to it, conversion to `Int` won't be necessary.

> 30 min is enough?

Typical installer size is about 100 Mb. If download speed is 64 kilobytes per second (I think that's a reasonable lower bound), they will download installer in 27 minutes. So 30 minutes should be enough.
But actually in 99% cases the limit should be about 5 minutes (assuming more or less decent internet connection).
So I would use a bit more complex logic: try with limit ≈ 5 min and then ≈ 30 minutes.